scripts/invoke.py

```diff
@@ -47,6 +47,11 @@ def main():
             print('--max_loaded_models must be >= 1; using 1')
             args.max_loaded_models = 1
 
+    # alert - setting a global here
+    Globals.root = os.path.expanduser(args.root_dir or os.environ.get('INVOKEAI_ROOT') or '.')
```

Comment:
Can arg parser read from environment variables as well? I'm used to config libraries that load configuration from multiple layered sources and then just provide a flatted config object.

Reply:
It could. I just wanted to make it clear what the precedence was. Can fix it if you feel this makes the code more brittle.